docs/manual/performance/cronjobs.en.md

```diff
@@ -0,0 +1,31 @@
+---
+title: 'Cronjob Framework'
+description: 'Setting up the Contao Cronjob framework.'
+aliases:
+    - /en/performance/cronjobs/
+---
+
+Contao ships with a cronjob framework by default. In short, this allows developers to register cronjobs 
```

Comment:
```suggestion
Contao ships with a cronjob framework by design. In short, this allows developers to register cronjobs 
```

Reply:
`default` is better than `design` in my opinion :)

Reply:
Two times by default.

Reply:
Changed.